pkgs/top-level/all-packages.nix

```diff
@@ -8105,6 +8105,8 @@ with pkgs;
 
   gmime = callPackage ../development/libraries/gmime { };
 
+  gmime3 = callPackage ../development/libraries/gmime/3.0.nix { };
```

Comment:
is it incompatible to gmime2?

Reply:
There might be breakage, yes. They have a porting document for migrating code from v2 API to v3 API:

https://github.com/jstedfast/gmime/blob/master/PORTING

Reply:
I will add an explanation to the commit message.